pkg/interop/convert/convert.go

```diff
@@ -6,11 +6,16 @@ func ToInteger(v interface{}) int {
 	return v.(int)
 }
 
-// ToByteArray converts it's argument to a ByteArray.
-func ToByteArray(v interface{}) []byte {
+// ToBytes converts it's argument to a Buffer VM type.
+func ToBytes(v interface{}) []byte {
```

Comment:
Then we need to update `customBuiltins`.

Reply:
Fixed.

Reply:
Oh, yeah.